### PR TITLE
Changed line 5 bool value to false

### PR DESCRIPTION
**Pull request by KaustubhMuhuri**

### CODE REVIEW

File: snowresponse.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
+   
     "result": {
         "parent": "",
-        "made_sla": "true",
+        "made_sla": "false",
```
Comment (Shreyash-procode) [inline snowresponse.json]: I dont need these changes

**Review by Shreyash-procode — COMMENTED** [review]
Change the line 5 again to true

**Review by Shreyash-procode — CHANGES_REQUESTED** [review]
Request you to change make it true again

File: snowresponse.json (hunk 1 above)
Comment (KaustubhMuhuri) [reply snowresponse.json]: Changing to true and making further changes

**Review by KaustubhMuhuri — COMMENTED** [review]
I have made changes

**Review by Shreyash-procode — APPROVED** [review]
Approved

**Pull request merged**